src/theory/PKIs.tex

```diff
@@ -150,15 +150,69 @@ \subsection{Hardening PKI}
 methods are used, DANE~\cite{rfc6698} and Certificate
 Pinning~\cite{draft-ietf-websec-key-pinning}. Google recently proposed
 a new system to detect malicious CAs and certificates called Certificate
-Transparency~\cite{certtransparency}.
+Transparency~\cite{certtransparency}. In addition, RFC 6844 describes Certification Authorization Records, a mechanism for domain name owners to signal which Certificate Authorities are authorized to issue certificates for their domain.
 
 % \subsubsection{DANE}
 % \label{sec:dane}
 
 % \subsubsection{Certificate Pinning}
 % \label{sec:certpinning}
 
+\subsection{Certification Authorization Records}
+\label{sec:caarecords}
 
+RFC 6844 describes Certification Authorization Records, a mechanism for domain name owners to signal which Certificate Authorities are authorized to issue certificates for their domain.
+ 
+When a CAA record is defined for a particular domain, it specifies that the domain owner requests Certificate Authorities to validate any request against the CAA record. If the certificate issuer is not listed in the CAA record, it should not issue the certificate.
+ 
+The RFC also permits Certificate Evaluators to test an issued certificate against the CAA record, but should exercise caution, as the CAA record may change during the lifetime of a certificate, without affecting its validity.
+ 
+CAA also supports an iodef property type which can be requested by a Certificate Authority to report certificate issue requests which are inconsistent with the issuer’s Certificate Policy.
+
+\subsubsection{Configuration of CAA records}
+\label{sec:pki:caarecords:configuration}
+ 
+BIND supports CAA records as of version 9.9.6.
+ 
+A CAA record can be configured by adding it to the zone file:
+
+\begin{lstlisting}
+$ORIGIN example.com
+       CAA 0 issue "ca1.org"
```

Comment:
I'd prefer a reserved domain as defined in [RFC 2606](https://tools.ietf.org/html/rfc2606).